packages/features/settings/outOfOffice/OutOfOfficeEntriesList.tsx

```diff
@@ -126,13 +126,14 @@ export const OutOfOfficeEntriesList = () => {
                       data-testid={`ooo-edit-${item.toUser?.username || "n-a"}`}
                       StartIcon="pencil"
                       onClick={() => {
+                        const offset = dayjs().utcOffset();
                         const outOfOfficeEntryData: BookingRedirectForm = {
                           uuid: item.uuid,
                           dateRange: {
-                            startDate: item.start,
-                            endDate: dayjs(item.end).subtract(1, "d").toDate(),
+                            startDate: dayjs(item.start).subtract(offset, "minute").toDate(),
+                            endDate: dayjs(item.end).subtract(offset, "minute").startOf("d").toDate(),
                           },
-                          offset: dayjs().utcOffset(),
+                          offset,
```

Comment:
`item.start` and `item.end` are timestamps, and we need to convert them to dates. For example,

From
- start: 2024-11-24 01:00:00+01:00
- end: 2024-11-25 00:59:59+01:00

To
- startDate: 2024-11-24
- endDate: 2024-11-24